python/cudf_polars/cudf_polars/callback.py

```diff
@@ -61,6 +64,8 @@ def execute_with_cudf(
     try:
         with nvtx.annotate(message="ConvertIR", domain="cudf_polars"):
             nt.set_udf(partial(_callback, translate_ir(nt)))
-    except exception:
+    except exception as e:
+        if "POLARS_VERBOSE" in Config.state(if_set=True):
```

Comment:
This doesn't work because if `POLARS_VERBOSE=0` this still triggers.
```suggestion
        if bool(int(os.environ.get("POLARS_VERBOSE", 0)):
```